exporter/exporter.go

```diff
@@ -312,10 +315,14 @@ func NewExporter(ctx context.Context, target, uri, profile, model string, exclud
 				handle(&exp, MEMORY_SUMMARY, STORAGEBATTERY)))
 
 		// DIMM endpoints array
-		dimms, err = getDIMMEndpoints(exp.url+sysEndpoints.systems[0]+"Memory/", target, retryClient)
+		if sysResp.Memory.URL == "" && len(sysEndpoints.systems) > 0 {
```

Comment:
```		if sysResp.Memory.URL != "" {
			dimms, err = getDIMMEndpoints(exp.url+sysResp.Memory.URL, target, retryClient)
			if err != nil {
```
Try this instead so that we do not get false error log in the absence of endpoint.